components/brave_new_tab_ui/containers/newTab/gridSites.tsx

```diff
@@ -34,101 +33,70 @@ interface Props {
   onShowEditTopSite: (targetTopSiteForEditing?: NewTab.Site) => void
 }
 
-interface State {
-  isDragging: boolean
+function TopSitesPage (props: Props & { maxGridSize: number, page: number }) {
+  const start = props.page * props.maxGridSize
+  const items = props.gridSites.slice(start, start + props.maxGridSize)
+  const { active } = useDndContext()
+  return <List>
+    {items.map((siteData) => (
+      <GridSiteTile
+        key={siteData.id}
+        actions={props.actions}
+        siteData={siteData}
+        onShowEditTopSite={props.onShowEditTopSite}
+        // User can't change order in "Most Visited" mode
+        // and they can't change position of super referral tiles
+        disabled={siteData.defaultSRTopSite || !props.customLinksEnabled}
```

Comment:
This property seems poorly named when actually it seems to actually control `isSortable`. The button / tile will still click and isn't really disabled like this implies. I know this was already named like this before this PR, so if you can improve the name whilst you're in it that would be a good readability improvement.

Reply:
:+1: good call